make_dashboards/templates/internal/js/dashboard.js

```diff
@@ -63,15 +63,23 @@ $(function () {
         
         // Middle Row - Queue plots
         // Max - 5 * pulse
+        //
+        sequencing_q_subtext=pl['miseq_sequencing_queue_l']+'<span style="color:#DF5353">M</span>,'+
+            pl['hiseq_sequencing_queue_l']+'<span style="color:#55BF3B">H</span>,'+
+            pl['hiseqX_sequencing_queue_l']+'<span style="color:#7cb5ec">X</span> lanes in queue';
         make_queue_plot('#lp_queue',      pl_l['library_prep'],   pl['library_prep_queue'],   pl['library_prep_queue']+' samples in queue');
-        make_queue_plot('#seq_queue',     pl_l['sequencing'],     pl['sequencing_queue'],     pl['sequencing_queue']+' lanes in queue');
+        make_queue_plot('#seq_queue',     pl_l['sequencing_queue'],     [pl['miseq_sequencing_queue_l'], pl['hiseq_sequencing_queue_l'], pl['hiseqX_sequencing_queue_l']], sequencing_q_subtext);
         make_queue_plot('#bioinfo_queue', pl_l['bioinformatics'], pl['bioinformatics_queue'], pl['bioinformatics_queue']+' lanes in queue');
         
         // Middle Row - Balance plots
+        sequencing_subtext=pl['miseq_sequencing_l']+'<span style="color:#AF2323">M</span>,'+
+            pl['hiseq_sequencing_l']+'<span style="color:#258F0B">H</span>,'+
+            pl['hiseqX_sequencing_l']+'<span style="color:#4c85bc">X</span> lanes in progress';
         make_balance_plot('#rc_finished_balance', pl_l['initial_qc_lanes'],   pl['initial_qc_lanes'],   undefined, pl['initial_qc_lanes']+' lanes in progress');
+        console.log(pl['initial_qc_samples']);
         make_balance_plot('#rc_balance',          pl_l['initial_qc_samples'], pl['initial_qc_samples'], undefined, pl['initial_qc_samples']+' samples in progress');
         make_balance_plot('#lp_balance',          pl_l['library_prep'],       pl['library_prep'],       undefined, pl['library_prep']+' samples in progress');
-        make_balance_plot('#seq_balance',         pl_l['sequencing'],         pl['sequencing'],         undefined, pl['sequencing']+' lanes in progress');
+        make_balance_plot('#seq_balance',         pl_l['sequencing']/4,         [pl['miseq_sequencing_l'], pl['hiseq_sequencing_l'], pl['hiseqX_sequencing_l']],         undefined, sequencing_subtext);
```

Comment:
At most, we can have 97 lanes running. The limit, 80, is the limit between
red and green. two thirds of the plot would never be used, but if I change
the limit, I think it affects other things. Also, Bahram wanted it changed,
so...
 Going to fix the logs

On Thu, Sep 1, 2016 at 2:06 PM, Phil Ewels <notifications@github.com> wrote:

> In make_dashboards/templates/internal/js/dashboard.js
> <https://github.com/NationalGenomicsInfrastructure/NGI_dashboards/pull/36#discussion_r77160892>
> :
>
> >          make_balance_plot('#rc_balance',          pl_l['initial_qc_samples'], pl['initial_qc_samples'], undefined, pl['initial_qc_samples']+' samples in progress');
> >          make_balance_plot('#lp_balance',          pl_l['library_prep'],       pl['library_prep'],       undefined, pl['library_prep']+' samples in progress');
> > -        make_balance_plot('#seq_balance',         pl_l['sequencing'],         pl['sequencing'],         undefined, pl['sequencing']+' lanes in progress');
> > +        make_balance_plot('#seq_balance',         pl_l['sequencing']/4,         [pl['miseq_sequencing_l'], pl['hiseq_sequencing_l'], pl['hiseqX_sequencing_l']],         undefined, sequencing_subtext);
>
> I don't understand pl_l['sequencing']/4 - is that not the name of the div
> to plot it in?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/NationalGenomicsInfrastructure/NGI_dashboards/pull/36/files/e6bda944c0764d53bb2c0a43764fc54201f113ac#r77160892>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AHLL0PcYU1Ux_r5moht1nzRtvcfAUSPtks5qlr_QgaJpZM4JdohU>
> .
>